packages/expo-ads-admob/android/src/main/java/expo/modules/ads/admob/AdMobInterstitialAdModule.java

```diff
@@ -136,6 +137,15 @@ public void run() {
   }
 
   private void recreateInterstitialAdWithAdUnitID(String adUnitID) {
+    if (mActivityProvider.getCurrentActivity() == null) {
+      if (mRequestAdPromise != null) {
+        mRequestAdPromise.reject("E_AD_INTERNAL_ERROR", "Currect activity is null.", null);
```

Comment:
Let's make error's code more clear 😉 
```suggestion
        mRequestAdPromise.reject("ERR_NO_CURRENT_ACTIVITY", "Currect activity is null.", null);
```